aggregator/src/main/scala/weco/concepts/aggregator/secrets/SecretsResolver.scala

```diff
@@ -0,0 +1,25 @@
+package weco.concepts.aggregator.secrets
+
+import grizzled.slf4j.Logging
+
+/*
+ * Selector to govern how to resolve secrets in config files.
+ * Do not use AWSDefault in production, as it can be very slow.
+ * When running on Lambda, use AWSEnvironment
+ */
+object SecretsResolver extends Logging {
+
+  def apply(key: String): Seq[String] => Map[String, String] = {
```

Comment:
Any sense in making this a sealed trait (like a union type) instead of a string?

Reply:
Possible, but it's chosen from a string in the config file.  I suppose Ficus can probably handle that case class name -> case class conversion, but I didn't think to look